mysql-test/suite/tianmu/r/delete.result

```diff
@@ -261,13 +257,9 @@ insert into t2 values (1,1),(2,2),(3,3);
 insert into t3 values (1,1),(2,1),(1,3);
 select * from t1,t2,t3 where t1.a=t2.a AND t2.b=t3.a and t1.b=t3.b;
```

Comment:
we need more  `and`, `or` condition expressions(mainly complex expressions). such as ` A and B or C` or `(a and b) or c`, etc.

Reply:
Yes, but now this block is still being modified. Only after the modification is finished and the result is correct can the verification be carried out